### ISSUE TITLE
Can't scroll down with arrow keys on autosuggestion bar 

### ISSUE THREAD

**Issue by josiah34** [withfig/fig#1716]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and tested again (tell us if that fixed it)

## Issue 
Can't seem to scroll down with my keyboard directional arrow keys on the autosuggestion bar as shown in video. 
[Screencast from 08-23-2022 08:21:36 PM.webm](https://user-images.githubusercontent.com/25124463/186288361-6c2e0b6a-e5f5-459b-aa75-3d329660cf73.webm)

### Environment

```bash
### Details:
|OS|Fig|Shell|
|-|-|-|
|Linux 5.15.0-46-generic|fig_desktop 2.3.10
|/usr/bin/zsh|
fig diagnostic

# Fig Diagnostics
## Fig details:
  - Fig version: 
## Hardware Info:
  - Model Name: cpu0
  - Model Identifier: AuthenticAMD
  - Chip: AMD Ryzen 5 5600X 6-Core Processor
  - Cores: 6
  - Memory: 16681906 KB
## OS Info:
  - Linux 5.15.0-46-generic
## Environment:
  - User Shell: Unknown UserShell
  - Current Directory: /home/code
  - CLI Installed: true
  - Executable Location: /usr/bin/fig
  - Current Window ID: 
  - Active Process: 
  - Terminal: <unknown>
  - Environment Variables:
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=5.1.1
    - PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/code/.local/bin:/home/code/.fig/bin
    - TERM=xterm-256color
    - TERM_SESSION_ID=44a76be0-e306-4cdc-bfa7-2743c38db66c
    - FIG_PID=9790
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: Unknown command
  - Hyper: Unknown command
  - Visual Studio Code: Unknown command
  - Docker:
```

**Comment by grant0417** [comment]
Hey @josiah34, what terminal are you using? Also do any other keys work (enter, tab, etc)?

**Comment by josiah34** [comment]
> Hey @josiah34, what terminal are you using? Also do any other keys work (enter, tab, etc)?

I'm using gnome-terminal. The suggestions appear but I'm unable to scroll down the list with the arrow keys. The mouse scroll works but doesnt highlight the suggestion so I dont get the description if I'm writing flags on commands. Pressing enter still works but only selects the first option since I cant use the arrows.

**Comment by grant0417** [comment]
Hey @josiah34, if you are available to debug this issue here is a link https://calendly.com/grant-fig/linux-debugging

**Comment by igornfaustino** [comment]
Same problem here! I'm also using gnome-terminal.
From what I've tested, this issue is only happening to me when I'm using zsh (I'm using it with oh-my-zsh and the powerlevel10k theme)
When I tested it with bash, everything seemed to work normally

**Comment by josiah34** [comment]
I'm also using ZSH with oh-my-zsh and powerlevel10k

**Comment by Fumler** [comment]
I use kitty terminal with zsh (prezto instead of oh-my-zsh) and experience same problem (I _do not._ use powerlevel though) Works fine in bash.

Enabled prezto plugins:
```
 zstyle ':prezto:load' pmodule \
   'environment' \
   'terminal' \
   'editor' \
   'history' \
   'syntax-highlighting' \
   'autosuggestions' \
   'directory' \
   'spectrum' \
   'utility' \
   'completion' \
   'git' \
```

(I know kitty isn't officially supported yet but this problem seems to be unrelated)

**Comment by grant0417** [comment]
I believe this issue should be fixed with the most recent version `2.3.17`

**Comment by squili** [comment]
Closing as this is fixed
